overlord/snapstate/storehelpers.go

```diff
@@ -241,6 +276,11 @@ func currentSnaps(st *state.State) ([]*store.CurrentSnap, error) {
 		return nil, err
 	}
 
+	if len(snapStates) == 0 {
+		// no snaps installed, do not bother any further
+		return nil, nil
+	}
+
```

Comment:
was there a specific reason to do this? it seems fine but a bit unrelated atm

Reply:
It's more of a drive-by change.